src/librustc_mir/interpret/memory.rs

```diff
@@ -56,12 +57,12 @@ pub struct Memory<'a, 'mir, 'tcx: 'a + 'mir, M: Machine<'a, 'mir, 'tcx>> {
     /// the wrong type), so we let the machine override this type.
     /// Either way, if the machine allows writing to a static, doing so will
     /// create a copy of the static allocation here.
-    alloc_map: M::MemoryMap,
+    pub(super) alloc_map: M::MemoryMap,
```

Comment:
Hm, now I just hope we won't accidentally bypass the memory API outside of interning. :/

Reply:
I tried exposing it just to interning, that didn't work. I can make `intern.rs` a submodule of `memory.rs` if you prefer

Reply:
Let's mark that as an issue or FIXME for later?